Modules/Projects/Controllers/ProjectsController.php

```
@@ -59,28 +63,56 @@ public function index(int $page = 0): \Illuminate\View\View
      */
     public function form(?int $id = null)
```

PHPMD (2.15.0)

Avoid variables with short names like $id. Configured minimum length is 3. (undefined)

(ShortVariable)